datafusion/physical-expr/src/aggregate/count_distinct.rs

```diff
@@ -83,6 +83,11 @@ macro_rules! float_distinct_count_accumulator {
     }};
 }
 
+/// Returns the estimated number of hashbrown hashtables is likely to come up again
```

Comment:
```suggestion
/// Returns the estimated number of hashbrown hashtables.
```

Also, can someone explain the reasoning behind this formula? Is it based on the hashbrown source code (in which case a link to the code would be nice)? CC @korowa

Reply:
Bucket number calculation is a simplified version of [capacity_to_buckets](https://github.com/rust-lang/hashbrown/blob/f2e62124cd947b5e2309dd6a24c7e422932aae97/src/raw/mod.rs#L195) from hashbrown, and I'd agree that it might be helpful to mention it.

Reply:
More context is here: https://github.com/apache/arrow-datafusion/pull/8721#discussion_r1441672719